js/client-chat.js

```diff
@@ -569,7 +569,27 @@
 			case 'query':
 				this.add('This is a PS system command; do not use it.');
 				return false;
-
+			case 'disablenotifs':
+			case 'enablenotifs':
+			case 'enablenotifications':
+			case 'disablenotifications':
+				var command = cmd.toLowerCase();
+				if (command.includes('disable')) {
+					if (Dex.prefs('nonotifs')) {
+						this.add("|error|You already have notifications disabled.");
+						return false;
+					}
+					Dex.prefs('nonotifs', true);
+					this.add("You will now not receive notifications.");
+				} else if (command.includes('enable')) {
+					if (!Dex.prefs('nonotifs')) {
+						this.add("|error|You are already receiving notifications.");
+						return false;
+					}
+					Dex.prefs('nonotifs', false);
```

Comment:
This doesn't do anything, does it?

Reply:
Huh, when did everywhere start using Dex.prefs?

Reply:
> This doesn't do anything, does it?

It does, in fact, set the value to `false` in storage. Dunno about the second part.

Reply:
Saw your commit. Changing now.